src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.tsx

```diff
@@ -302,14 +304,24 @@ export const VenueRoomsEditor: React.FC<VenueRoomsEditorProps> = ({
         handleHeight
         onResize={(width, height) => setImageDims({ width, height })}
       />
-      <img
-        alt="draggable background "
-        style={{
-          width: "100%",
-          ...backgroundImageStyle,
-        }}
-        src={backgroundImage}
-      />
+      {!backgroundImage ? (
+        <div className="Container__background-image-placeholder">
+          <Centered>
```

Comment:
This will introduce a lot of new and unnecessary divs. It would be easier if you move the styles in the `global.scss` and just pass them in the `className`

Reply:
The other way around is to pass the parent className `Container__background-image-placeholder` in here as a prop. And it would still be a single div.

Reply:
I removed the `div` inside the `Centered` component here. 
The div on line 308 here is necessary as it renders the rounded rectangle with a background as in designs.